src/container/srv_target.c

```diff
@@ -567,7 +567,7 @@ ds_cont_tgt_open_handler(crt_rpc_t *rpc)
 		DP_CONT(in->toi_pool_uuid, in->toi_uuid), rpc,
 		DP_UUID(in->toi_hdl));
 
-	rc = dss_task_collective(cont_open_one, in, 0);
+	rc = dss_thread_collective(cont_open_one, in, 0);
```

Comment:
if you move resync_dtx outside of ds_cont_local_open(), you do not need change this?

Reply:
Move dtx_resync() from ds_cont_local_open() to where? We need to resync DTX status when open the container. If use dss_task_collective(), it will hung there.

Reply:
why it has to be in open(), I thought as long as it is before normal rebuild, it should be ok? or I miss sth?

Reply:
It is not must be in open, but for non-rebuild object case, when the server restarts, we also need to re-sync former committable DTX status, the open container is a good point to do such work.